src/main/cc/wfa/virtual_people/common/field_filter/utils/field_util.cc

```diff
@@ -0,0 +1,131 @@
+// Copyright 2021 The Cross-Media Measurement Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "wfa/virtual_people/common/field_filter/utils/field_util.h"
+
+#include "absl/memory/memory.h"
+#include "absl/status/status.h"
+#include "absl/strings/str_cat.h"
+#include "absl/strings/str_split.h"
+#include "absl/strings/string_view.h"
+#include "google/protobuf/descriptor.h"
+
+namespace wfa_virtual_people {
+
+absl::Status GetFieldFromProto(
+    const google::protobuf::Descriptor* descriptor,
+    absl::string_view full_field_name,
+    std::vector<const google::protobuf::FieldDescriptor*>* field_descriptors) {
+  field_descriptors->clear();
```

Comment:
Looks like this is assuming that field_descriptors is non-null. Could you do one of these two things?

1. Just return an absl::StatusOr<std::vector<const google::protobuf::FieldDescriptor*>> (preferred)
2. Pass `field_descriptors` by mutable reference instead of by pointer?

Reply:
Done.